bin/clever.js

```diff
@@ -226,6 +226,12 @@ function run () {
       aliases: ['f'],
       description: 'Force deploy even if it\'s not fast-forwardable',
     }),
+    watchDeploy: cliparse.option('watch', {
+      aliases: ['w'],
+      default: true,
+      description: 'Should watch a deployment logs',
```

Comment:
`watch` is not meaningfu for me, 
What about `wait` 
with description `Wait for deployment ends`

Reply:
We decided to change the naming to `exit-on`